src/components/cylc/Header.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-list-tile-action-text class="pt-3 pb-3 c-header">
+    <v-layout align-center justify-center row wrap fill-height>
+      <div class="mb-2">
+        <svg version="1.1" viewBox="0 0 655 260" xmlns="http://www.w3.org/2000/svg" xmlns:osb="http://www.openswatchbook.org/uri/2009/osb">
+          <g transform="translate(292.53 -49.505)">
```

Comment:
[inkscape rant]

Gotta love Inkscape, hey lets define a document 655x260 but then wrap everything using a completely unecessary and totally arbitrary transform of 292.53x-49.505, that'll make all the number really nice!

[/inkscape rant]

Good job exporting with 5sf precision though!

Reply:
First time I did it for the web, just followed a tutorial. :relieved:

Reply:
Nothing you've done, Just Inkscape being daft!